src/graphs.jl

```diff
@@ -88,8 +88,10 @@ end
     add_vertex!(meta_graph, label, data)
 
 Add a vertex to MetaGraph `meta_graph` with label `label` having metadata `data`.
+If the `VertexData` type of `meta_graph` is `Nothing`, `data` can be omitted.
```

Comment:
I would rather document the actual new method (and add some tests for it too?)

Reply:
My motivation for this was mostly to keep the `help>` output not too verbose: if we add a separate doc-string, it balloons the output and forces you to scroll if you're on a narrow terminal. E.g., the current output of `help> add_edge!` is:
![image](https://user-images.githubusercontent.com/26799210/222670918-19607a33-dca8-4784-88a9-deec26990890.png)

I can change it if you prefer but wanted to point out this motivation.

Reply:
ok that makes sense, let's keep it that way